### PR TITLE
First complete iteration

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
This is the first complete iteration on VTEX IO Landing Page.

#### How should this be manually tested?

https://victorhugo--storecomponents.myvtex.com/dev/io

#### Screenshots or example usage

Screen Shot 2019-04-09 at 22 19 02

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: messages/context.json [hunk 1]
```diff
@@ -18,5 +18,40 @@
...
+  "evolution.focus": "Focus on what matters",
+  "evolution.evolution": "Seamless evolution.",
+  "evolution.perspective": "VTEX IO delivers a new paradigm for cloud software development that’s heavily integrated into the environment where your code will actually run. This shift in perspective unlocks new capabilities that were impossible on the traditional, local development model.",
+  "evolution.vtexio": "VTEX IO",
+  "evolution.vtexio.description": "It's our serverless development platform, designed to help you create scalable, production-ready web applications – and never have to worry about any infrastructure complexity again.",
+  "features.title": "Powerful tools to power-up your solutions.",
+  "features.title.small": "Make the most out of your work",
+  "features.cloud": "Works on the cloud",
+  "features.cloud.description": "It's the end of the mismatch between local and production environments. No code runs on your computer: The source gets synced to the cloud, where it is built and deployed in real-time.",
+  "features.setupless": "Setupless",
```
Comment (rafarubim) [inline messages/context.json]: Does this word actually exist? 😛

File: messages/en.json [hunk 2]
```diff
@@ -18,5 +18,40 @@
...
+  "evolution.focus": "Focus on what matters",
+  "evolution.evolution": "Seamless evolution.",
+  "evolution.perspective": "VTEX IO delivers a new paradigm for cloud software development that’s heavily integrated into the environment where your code will actually run. This shift in perspective unlocks new capabilities that were impossible on the traditional, local development model.",
+  "evolution.vtexio": "VTEX IO",
+  "evolution.vtexio.description": "It's our serverless development platform, designed to help you create scalable, production-ready web applications – and never have to worry about any infrastructure complexity again.",
+  "features.title": "Powerful tools to power-up your solutions.",
+  "features.title.small": "Make the most out of your work",
+  "features.cloud": "Works on the cloud",
+  "features.cloud.description": "It's the end of the mismatch between local and production environments. No code runs on your computer: The source gets synced to the cloud, where it is built and deployed in real-time.",
+  "features.setupless": "Setupless",
```
Comment (rafarubim) [inline messages/en.json]: Also here

File: messages/context.json [hunk 3]
```diff
@@ -18,5 +18,40 @@
   "howitworks.abtesting.tests": "Perform tests on every aspect of your business to learn what works best. Use the isolated Test Workspaces to iterate faster, increase sales, and raise conversion rates.",
   "howitworks.storeframework": "Store Framework",
   "howitworks.storeframework.build": "Build faster, grow better",
-  "howitworks.storeframework.develop": "Reduce loading time, improve usability, and make the best of SEO. Developing scalable components with a comprehensive, easy-to-use toolset, you can build stores faster than ever."
+  "howitworks.storeframework.develop": "Reduce loading time, improve usability, and make the best of SEO. Developing scalable components with a comprehensive, easy-to-use toolset, you can build stores faster than ever.",
```
Comment (rafarubim) [inline messages/context.json]: I noticed the context.json is just being used as a copy of en.json. I understand you probably don't mean to change it in this PR, so this shouldn't block the review 😬

Comment (rafarubim) [inline messages/context.json]: I noticed the context.json is just a copy of en.json. I understand that you don't mean to change it yet so this won't block my review 😬

File: messages/en.json (hunk 2 above)
Comment (rafarubim) [inline messages/en.json]: I'm almost sure it doesn't actually exist hahaha

File: react/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+              </li>
+            ))}
+          </ul>
+        )
+      })}
+    </div>
+    <p className="fw7">© VTEX, 2018</p>
+    <div className="flex flex-wrap center mv7">
+      {VTEXOffices.Offices.map(office => (
+        <div key={office.short} className="ph4" style={{ width: `${20}%` }}>
```
Comment (rafarubim) [inline react/Footer.tsx]: I think w-20 tachyons class could be used here

File: react/StartBuilding.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      <img src={IllustrationImage} alt="" className="mb0" />
+    </div>
+    <div className="w-50 min-h-100 flex flex-column justify-between items-center">
+      <p className="t-heading-1 w-50 normal h-50">
+        Start building the future of commerce.
+      </p>
+      <div className="w-50 h-50 flex flex-column justify-center">
+        <p className="t-body mb7">
+          Build apps with cutting-edge technology, meet the needs of world-class
+          brands and expand the possibilities of retail.
```
Comment (rafarubim) [inline react/StartBuilding.tsx]: You might want to internationalize here

File: react/StartBuilding.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        </p>
+        <div className="w-75">
+          <Button variation="secondary">
+            <p className="action-secondary">BECOME A PARTNER</p>
```
Comment (rafarubim) [inline react/StartBuilding.tsx]: Here too

File: messages/en.json (hunk 2 above)
Comment (rafarubim) [reply messages/en.json]: Duplicated comment again, I'm on the tablet  and this time couldn't even edit it lol

File: messages/context.json (hunk 1 above)
Comment (victorhmp) [reply messages/context.json]: HAHAHAHA
Pretty sure it doesn't... but I guess it is kinda acceptable 😂
I'll ask the designers and branding team if we should really use this word

File: messages/context.json (hunk 3 above)
Comment (victorhmp) [reply messages/context.json]: Yep! This will be fixed in a future PR, I did this to iterate faster.

File: react/StartBuilding.tsx (hunk 5 above)
Comment (victorhmp) [reply react/StartBuilding.tsx]: Yep! Good catch!

File: react/StartBuilding.tsx (hunk 6 above)
Comment (victorhmp) [reply react/StartBuilding.tsx]: Thanks hahaha those slipped right past me

File: react/Footer.tsx (hunk 4 above)
Comment (victorhmp) [reply react/Footer.tsx]: Yep! Much better

**Pull request merged**